src/installer/installer.go

```diff
@@ -95,14 +98,17 @@ func (i *installer) InstallNode() error {
 
 	i.UpdateHostInstallProgress(models.HostStageStartingInstallation, i.Config.Role)
 	i.Config.Device = i.ops.EvaluateDiskSymlink(i.Config.Device)
-	err := i.cleanupInstallDevice()
-	if err != nil {
-		i.UpdateHostInstallProgress(models.HostStageStartingInstallation, fmt.Sprintf("Could not clean install device %s. The installation will continue. If the installation fails, clean the disk and try again", i.Device))
-		// Do not change the phrasing of this error message, as we rely on it in a triage signature
-		i.log.Errorf("failed to prepare install device %s, err %s", i.Device, err)
+	if i.DryRunEnabled || i.Config.SkipInstallationDiskCleanup {
```

Comment:
Maybe better to wrap those if-esle in function ? and to leave it same way it was?